File: selling-used-items-app-frontend/src/app/model/report.ts
export class Report {
    id: number = 0;
    userId: number = 0;
    reason: string = '';
  
    constructor(
      id: number = 0,
      userId: number = 0,
      reason: string = ''
    ) {
      this.id = id;
      this.userId = userId;
      this.reason = reason;
    }
  }
  

File: selling-used-items-app-backend/selling-used-items-app-backend/Model/Report.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace selling_used_items_app_backend.Model
{
    public class Report
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int id { get; set; }

        [ForeignKey("User")]
        public int userId { get; set; }

        public String reason { get; set; }
    }
}